contracts/varatube/tests/utils.rs

```diff
@@ -0,0 +1,188 @@
+        assert!(!res.main_failed());
+        varatube
+    }
+    fn register_subscription(
+        &self,
+        from: u64,
+        currency_id: ProgramId,
+        period: Period,
+        with_renewal: bool,
+        error: Option<Error>,
+    ) {
+        let program_id_bytes: [u8; 32] = currency_id.into();
+        let result = self.send(
+            from,
+            Actions::RegisterSubscription {
+                currency_id: program_id_bytes.into(),
+                period,
+                with_renewal,
+            },
+        );
+        if let Some(error) = error {
+            assert!(result.contains(&(from, error.encode())));
+        } else {
+            let expected_reply: Result<Reply, Error> = Ok(Reply::SubscriptionRegistered);
+            assert!(result.contains(&(from, expected_reply.encode())));
+        }
+    }
+
+    fn update_subscription(&self, from: u64, subscriber: u64, error: Option<Error>) {
+        let result = self.send(
+            from,
+            Actions::UpdateSubscription {
+                subscriber: subscriber.into(),
+            },
+        );
+        if let Some(error) = error {
+            assert!(result.contains(&(from, error.encode())));
+        } else {
+            let expected_reply: Result<Reply, Error> = Ok(Reply::SubscriptionUpdated);
+            assert!(result.contains(&(from, expected_reply.encode())));
+        }
+    }
+
+    fn cancel_subscription(&self, from: u64, error: Option<Error>) {
+        let result = self.send(from, Actions::CancelSubscription);
+        if let Some(error) = error {
+            assert!(result.contains(&(from, error.encode())));
+        } else {
+            let expected_reply: Result<Reply, Error> = Ok(Reply::SubscriptionCancelled);
+            assert!(result.contains(&(from, expected_reply.encode())));
+        }
+    }
+
+    fn add_token_data(
+        &self,
+        from: u64,
+        currency_id: ProgramId,
+        price: Price,
+        error: Option<Error>,
+    ) {
+        let program_id_bytes: [u8; 32] = currency_id.into();
+        let result = self.send(
+            from,
+            Actions::AddTokenData {
+                token_id: program_id_bytes.into(),
+                price,
+            },
+        );
+        if let Some(error) = error {
+            assert!(result.contains(&(from, error.encode())));
+        } else {
+            let expected_reply: Result<Reply, Error> = Ok(Reply::PaymentAdded);
+            assert!(result.contains(&(from, expected_reply.encode())));
+        }
+    }
+
+    fn get_subscriber_data(&self, subscriber: u64) -> Option<SubscriberData> {
+        let state: StateReply = self
+            .read_state(StateQuery::GetSubscriber(subscriber.into()))
+            .expect("Unable to read varatube state");
+        if let StateReply::SubscriberData(subscriber_data) = state {
+            subscriber_data
+        } else {
+            gstd::panic!("Wrong received reply");
```

Comment:
Why not `std::panic`?
```suggestion
            panic!("Wrong received reply");
```